contracts/common/Multicall.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Mostly taken from Uniswap/v3-periphery repository, with payable removed and solidity version
+// adjustments
+pragma solidity 0.7.3;
+pragma experimental "ABIEncoderV2";
+import "./MetaTransactionMsgSender.sol";
+
+/// @title Multicall
+/// @notice Enables calling multiple methods in a single call to the contract
```

Comment:
Not consistent with our style to have comments above contracts...

Reply:
I'm not exactly sure what you're after here?

Reply:
I'm saying it's stylistically inconsistent to have natspec comments above contract definitions

Reply:
Ah, I had parsed your original comment as "Not consistent with [our style to have comments above contracts...]" and was confused :sweat_smile:

Reply:
Would you like me to remove them from other places they are present as well (i.e. IMulticall, IRecovery, ContractRecovery)?

Reply:
I suppose that is what I'm saying, although now I feel like a nitpick. Do you have strong feelings about the stylistic consistency?

Reply:
Nope, I just copied a random contract that happened to already have them.

Reply:
Ok. I'm not going to ask you to go around editing unrelated files, so let's leave it at this for now.